assets/terraform-modules/packet/flatcar-linux/kubernetes/workers/cl/worker.yaml.tmpl

```diff
@@ -421,6 +434,35 @@ storage:
           done
           echo "$record.$zone is available on all nameservers"
           exit 0
+    - path: /opt/wait-for-bgp
+      filesystem: root
+      mode: 0544
+      contents:
+        inline: |
+          #!/bin/bash
+          set -o pipefail
+          max_attempts=3600
+          echo "Polling metadata service for BGP metadata"
+          counter=0
+          while [[ $counter -lt $max_attempts ]]; do
+              out=$(curl -s -f --connect-timeout 5 \
```

Comment:
What if BGP is not enabled for some reason on the host? Are we able to differentiate somehow situation when it's not enabled completely from not _yet_ enabled?

Reply:
Oh, good catch. We do have conditionals in the kubelet template for when BGP is disabled, however we must also disable this script, otherwise kubelet will never start.

Reply:
Ah, hehe. I meant more like a bug in Terraform code. I forgot that we actually have an option to disable BGP for workers :)

Reply:
In case it helps, it is here: https://github.com/kinvolk/lokomotive/blob/master/pkg/platform/packet/template.go#L207

Reply:
Added a conditional around enabling `wait-for-bgp.service`.